core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-reports/dot-experiments-reports.component.scss

```diff
@@ -42,12 +42,14 @@ $max-container-width: 1100px;
     color: $color-alert-green;
 }
 
-.prospect-winner {
-    font-weight: $font-weight-bold;
+.trophy {
+    fill: $color-accessible-text-green;
+    height: 20px;
+    min-width: 20px;
```

Comment:
Should we use `rem `values?

Reply:
In this case because is a SVG I think no, but maybe we put this in the html?